src/main.ts

```diff
@@ -262,6 +263,26 @@ export class Purchases {
     );
   }
 
+  /**
+   * Retrieves a specific offering by its identifier.
+   * @param offeringIdentifier - The identifier of the offering to retrieve.
+   * @param params - The parameters object to customise the offerings fetch. Check {@link GetOfferingsParams}
+   */
+  public async getOffering(
+    offeringIdentifier: string,
```

Comment:
I wonder if this could instead be a parameter inside the `GetOfferingParams`... so we keep it to a single method...